src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllCodeFixSuggestedAction.FixAllCodeAction.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    internal partial class FixAllCodeFixSuggestedAction
+    {
+        private sealed partial class FixAllCodeAction : FixSomeCodeAction
```

Comment:
Existing code - renaming the outer type from `FixAllSuggestedAction` to `FixAllCodeFixSuggestedAction` and moved some code to the base type.

https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllSuggestedAction.FixAllCodeAction.cs